apps/console/src/features/core/models/config.ts

```diff
@@ -191,6 +191,10 @@ export interface UIConfigInterface extends CommonUIConfigInterface<FeatureConfig
      * `REMOTE` will fetch templates from the template management REST API.
      */
     applicationTemplateLoadingStrategy?: ApplicationTemplateLoadingStrategies;
+    /**
+     * Configuration to enable Google One Tap for specific tenants.
+    */
```

Comment:
```suggestion
     */
```

Reply:
This is fixed c331826f2c27a2b48167c75c536bc3b90f258c6d